### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/24.07/java%E7%9A%84asm%E5%BA%93%E6%B7%B1%E5%85%A5%E6%B5%85%E5%87%BA

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#56] [General]
# https://www.xiaogenban1993.com/blog/24.07/java%E7%9A%84asm%E5%BA%93%E6%B7%B1%E5%85%A5%E6%B5%85%E5%87%BA

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/24.07/java%E7%9A%84asm%E5%BA%93%E6%B7%B1%E5%85%A5%E6%B5%85%E5%87%BA

**Comment by Ak5mai** [comment]
膜拜大佬，视频讲的太通透了，一晚上给我学爽了：）

**Comment by anansite** [comment]
邮件已收到，我会尽快查看
